source/faq/faq.rst

```diff
@@ -123,115 +119,139 @@ Devices
+Consider, for instance, an actual device with temperature and humidity sensors;
+it can be represented into dojot as a device with two attributes (one for each
+sensor). We call this kind of device as *regular device* or by its
+communication protocol, for instance, *MQTT device* or *CoAP device*.
 
-We can also create devices which don’t directly correspond to their actual ones, for instance, 
-we can create one with higher level of information of temperature (*is becoming hotter* or 
-*is becoming colder*) whose values are inferred from temperature sensors of other devices. 
-This kind of device is called *virtual device*.
+We can also create devices which don’t directly correspond to their actual
+ones, for instance, we can create one with higher level of information of
+temperature (*is becoming hotter* or *is becoming colder*) whose values are
+inferred from temperature sensors of other devices. This kind of device is
+called *virtual device*.
 
 What is the relationship between this *device* and my actual device?
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-It is simple as it seems: the *regular device* for dojot is a mirror (digital twin) of 
-your actual device. You can choose which attributes are available for applications and 
-other components by adding each one of them at the device creation interface. If you don't 
-want some attributes to be available to applications or other elements, 
-just don't add them in dojot.
+It is simple as it seems: the *regular device* for dojot is a mirror (digital
+twin) of your actual device. You can choose which attributes are available for
+applications and other components by adding each one of them at the device
+creation interface. If you don't want some attributes to be available to
+applications or other elements, just don't add them in dojot.
 
 What are *virtual devices*? How are they different from the other one?
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-*Regular devices* are created to serve as a mirror (digital twin) for the actual devices 
-and sensors. A *virtual device* is an abstraction that models things that are not feasible 
-in the real world. For instance, let's say that a user has few smoke detectors in a laboratory, 
-each one with different attributes. 
-Wouldn't it be nice if we had one device called *Laboratory* that has one attribute *isOnFire*? 
-So, the applications could rely only on this attribute to take an action.
+*Regular devices* are created to serve as a mirror (digital twin) for the
+actual devices and sensors. A *virtual device* is an abstraction that models
+things that are not feasible in the real world. For instance, let's say that a
+user has few smoke detectors in a laboratory, each one with different
+attributes.
+
+Wouldn't it be nice if we had one device called *Laboratory* that has one
+attribute *isOnFire*? So, the applications could rely only on this attribute to
+take an action.
```

Comment:
> So, the applications could rely only on this attribute to take an action.

In a way that applications could rely ...

Reply:
Ok